AdyenCard/Utilities/CardType/CardBrand.swift

```diff
@@ -8,6 +8,19 @@ import Foundation
 
 /// Describes the Card brand.
 public struct CardBrand: Decodable {
+    
+    /// Indicates the requirement level of a field.
+    public enum RequirementPolicy: String, Decodable {
```

Comment:
it can be internal I think, its most probably not used by merchants since `cvcPolicy` was internal.

Reply:
Changing it :)